drivers/mpu9250/MPU9250.cpp

```diff
@@ -232,51 +125,62 @@ int MPU9250::mpu9250_init()
 	 */
 	//uint8_t samplerate_divider = 0;
 	//result = _writeReg(MPUREG_FIFO_EN, samplerate_divider);
-
 	//if (result != 0) {
 	//	DF_LOG_ERR("sample rate config failed");
 	//}
-
 	//usleep(1000);
-
 	result = _writeReg(MPUREG_CONFIG,
-			   BITS_DLPF_CFG_250HZ |
-			   BITS_CONFIG_FIFO_MODE_OVERWRITE);
+			BITS_DLPF_CFG_250HZ | BITS_CONFIG_FIFO_MODE_OVERWRITE);
```

Comment:
For next time, please try to minimize whitespace changes such as these. It makes reading the diff harder and in this case is not even better but rather depends on taste.